doc/api/querystring.md

```diff
@@ -68,9 +68,9 @@ For example, the query string `'foo=bar&abc=xyz&abc=123'` is parsed into:
 ```
 
 *Note*: The object returned by the `querystring.parse()` method _does not_
-prototypically extend from the JavaScript `Object`. This means that the
-typical `Object` methods such as `obj.toString()`, `obj.hasOwnProperty()`,
-and others are not defined and *will not work*.
+prototypically extend the JavaScript `Object`. This means that typical`Object`
+methods such as `obj.toString()`, `obj.hasOwnProperty()`, and others are not
+defined and *will not work*. As well `obj instanceof Object === false`.
```

Comment:
Can you try to use complete sentences in the documentation? Also, this isn’t that simple anyway. Other Contexts can have different `Object` objects anyway – the better advice is to avoid `instanceof` for builtins in JS code (i.e. please drop the last sentence here).